crates/stages/src/stages/sender_recovery.rs

```diff
@@ -93,7 +91,9 @@ impl<DB: Database> Stage<DB> for SenderRecoveryStage {
         for chunk in
             &tx_walker.chunks(self.commit_threshold as usize / rayon::current_num_threads())
         {
-            let tx = tx.clone();
+            // An _unordered_ channel to receive results from a rayon job
+            let (tx, rx) = mpsc::unbounded_channel();
```

Comment:
this can be a oneshot channel now, right?

Reply:
Yeah, I imagined this differently than it was done, indeed in this setup oneshot would be better. Will make modifications. I want to send the recovered sender right after it is calculated, so it can be saved to the database.